lib/DBDish/Pg/StatementHandle.pm6

```diff
@@ -94,7 +94,7 @@ method _row() {
                 $value = $!result.PQgetvalue($!current_row, $col);
                 if ct ~~ Array {
                     $value = _pg-to-array($value, ct.of);
-                } else {
+    			} elsif (ct.^name ne 'Any') {
```

Comment:
Maybe "ct === Any" rather than the string comparison.

Also wonky indentation :)